src/components/header-search-input/index.tsx

```diff
@@ -13,7 +13,9 @@ function HeaderSearchInput({
 }: HeaderSearchInputProps): React.ReactElement {
   return (
     <DevPopover
-      title="Work in progress"
+      buttonClassName="g-focus-ring-from-box-shadow-dark"
+      containerClassName={s.popoverContainer}
+      title="Single Product searching is currently a work in progress"
```

Comment:
Few changes here:

- `buttonClassName`: added the global focus ring helper so the focus style matches the others used in the header
- `containerClassName`: see my comment on the `.popoverContainer` class
- `title`: this is just a little more descriptive, but it'll be going away with beta anyways 😊